saleor/core/utils/random_data.py

```diff
@@ -470,7 +455,6 @@ def create_products_by_schema(placeholder_dir, create_images):
     assign_attribute_values_to_variants(
         types["attribute.assignedvariantattributevalue"]
     )
-    assign_attributes_to_pages(page_attributes=types["attribute.assignedpageattribute"])
```

Comment:
This wasn't doing anything as there is no `attribute.assignedpageattribute` in the `populatedb_data.json` file 🤷‍♀️